packages/discussions/migrations/sqls/20190530022452-fix-depth-down.sql

```diff
@@ -0,0 +1 @@
+SELECT 1;
```

Comment:
yes, because I can't undo counting the correct depth :smirk: (The value `depth` now is correct for old comments, but 0 for all new (after launch) ones).